CHANGELOG.md

```diff
@@ -1,6 +1,56 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-05-13
 
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
```

Comment:
@palma21 I know it's not technically linked with a particular release, but should we add a line to the new AKS blog to increase awareness on a good source for more info to go along with release notes and docs?

Reply:
Ah I missed a line on it. Never mind